db/migrate/20170217194936_add_properties_to_physical_servers.rb

```diff
@@ -0,0 +1,16 @@
+class AddPropertiesToPhysicalServers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_servers, :hostname, :string
+    add_column :physical_servers, :ipv_4_addresses, :string, :array => true, :default => []
+    add_column :physical_servers, :ipv_6_addresses, :string, :array => true, :default => []
+    add_column :physical_servers, :mac_addresses, :string, :array => true, :default => []
```

Comment:
A lot of this is duplicated from hardwares table (and subsequent tables).  We should probably leverage that.  Alternately, we may want to start using the computer_systems table, as its purpose is to abstract the multitude of tables like hardwares, operating_system, etc. 

@bdunne Thoughts here?  cc @blomquisg